translations/es.json

```
@@ -31,14 +31,25 @@
 		}
 	},
 	"expression": {
+		"variables": "Variables",
 		"title": "Crea una expresion",
-		"objects": "Objetos",
+		"mainObjects": "Mis objectos",
+		"wollokObjects": "Objectos en Wollok",
 		"literals": "Literales",
 		"messages": "Mensajes",
 		"aNumber": "Un numero",
 		"aString": "Un texto",
 		"true": "Verdadero",
 		"false": "Falso",
-		"null": "Nulo"
+		"null": "Nulo",
+		"notSeggestions": "No tenemos sugerencias aquí, usá el panel de búsqueda",
+		"search": {
+			"message": "Buscá algún mensaje",
+			"entity": "Buscá alguna entidad"
+		}
+	},
+	"sentence": {
+		"assignment": "asignacion",
```

```suggestion
		"assignment": "asignación",
```

Porfa escribamos bien esto